portal-web/test/functional/com/liferay/portalweb/macros/SystemSettings.macro

```diff
@@ -133,7 +133,7 @@ definition {
 	macro editTextAreaSetting {
 		Type(
 			key_label = ${settingName},
-			locator1 = "TextArea#ANY_LABELED",
+			locator1 = "TextArea#ANY_LABELED_OPTIONAL",
 			value1 = ${settingValue});
 	}
```

Comment:
Could you use something more specific here, maybe create a macro for the walkthrough? because this macro is used in many tests and it may be that this modification breaks the others.